azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/controlbar/ControlBarView.kt

```diff
@@ -101,8 +101,8 @@ internal class ControlBarView : ConstraintLayout {
 
     private val alwaysOffSelectedAccessibilityDelegate = object : AccessibilityDelegateCompat() {
         override fun onInitializeAccessibilityNodeInfo(
-            host: View?,
-            info: AccessibilityNodeInfoCompat?
+            host: View,
+            info: AccessibilityNodeInfoCompat
```

Comment:
would it be possible to fix this as separate PR if required as this is not part of chat